k8s/preview/unspec/xui-identity.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: xui
```

Comment:
xui ?

Reply:
yeah, we're currently using it on preview